share/github-backup-utils/ghe-backup-pages

```diff
@@ -75,7 +60,7 @@ for hostname in $hostnames; do
 
   # Transfer all data from the user data directory using rsync.
   ghe-rsync -avz \
-    -e "ssh -q $opts -p $port $ssh_config_file_opt -l $user" \
+    -e "ghe-ssh" \
```

Comment:
This is great ✨ . `$port` is currently derived from `ssh_port_part`, which [returns port 22 by default](https://github.com/github/backup-utils/blob/315bec8d8e1247b7ab1f9621e5fdd35672bfbe77/share/github-backup-utils/ghe-backup-config#L276-L278). Switching this to `ghe-ssh` will prevent some cases where `ghe-backup-pages` (and some other scripts) try to connect to the wrong port.